packages/js/src/workouts/components/ConfigurationWorkout.js

```diff
@@ -745,14 +639,15 @@ export function ConfigurationWorkout( { finishSteps, reviseStep, toggleWorkout,
 
 	// PROBABLY DELETE BETWEEN HERE....
 	const isStepperFinished = [
-		isStep1Finished,
 		isStep2Finished,
 		isStep3Finished,
 		isStep5Finished,
 	].every( Boolean );
 
+	/* Duplicate site representation, because in reality, the first step cannot be saved.
+	It's considered "finished" once at least the site representation has been done. */
 	const savedSteps = [
-		isStepFinished( "configuration", steps.optimizeSeoData ),
+		isStepFinished( "configuration", steps.siteRepresentation ),
 		isStepFinished( "configuration", steps.siteRepresentation ),
```

Comment:
I've tried to explain that in a (temporary) comment on lines 647-648:

The helper function inside the useState on line 658 (`getInitialActiveStepIndex`) looks for the first "false" in this array, and returns the index of that entry.

Since the indexation step is never actually really saved, but should be skipped if the user is already passed the siterepresentation step, we went with this solution.

I do agree it is a bit unclear, but we tried to avoid another refactor of the helper function.
Although on second thought, the helper function is pretty useless now that UX has ordered different behavior.

Let me think of a fix.